modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/index.js

```diff
@@ -12,6 +12,6 @@
  * details.
  */
 
-export {ClassicEditor} from './editor/ClassicEditor';
+export {default as ClassicEditor} from './editor/ClassicEditor';
```

Comment:
I think this might be a breaking change...

Reply:
I don't think so, because it was already broken, that way we will have the same behavior as the others, there is no export non-default in ClassicEditor, I think it was just used in taglib, we can add a non-default export also in ClassicEditor and remove it.